### ISSUE TITLE
Clair - There is no way to pass extra CA certificates to Clair pod

### ISSUE THREAD

**Issue by scouturier** [quay/quay-operator#179]
Currently, there is no documented ways to pass extra CA certificates to Clair pod, therefore if Quay is using a self-signed (with CA) certificate on it's route, Clair will not trust it unless the certificate is manually added to the Deployment.

Could be similar to the extraCaCert for quay:
```
  quay:
    configFiles:
      - secretName: quayconfigfile
        type: extraCaCert
```

**Comment by liveaverage** [comment]
Just ran into the same thing and incorrectly assumed extraCaCert definitions for clair would work. Like you, manually added the CA bundle directly to the Deployment (mounting at `$CLAIRDIR/certs/` per https://github.com/quay/clair-jwt/blob/master/generate_mitm_ca.rhel.sh#L14) and everything worked fine... but it'd be nice to see this added to the operator.

**Comment by sabre1041** [comment]
This feature is currently being worked on and should be available as part of the next release

**Comment by sabre1041** [comment]
Resolved as part of #187
